### ISSUE TITLE
Steam no small logo.

### ISSUE THREAD

**Issue by Durus-R** [Heroic-Games-Launcher/HeroicGamesLauncher#2639]
### Describe the bug

The game has no logo, when I add a game to steam, although it has a banner and a hero image in the bigpicture mode.

### Add logs

```text
(No log)
```

### Steps to reproduce

Click on "Add game to steam". Then, look into your steam library after restarting steam

### Expected behavior

The game should be listed. If you click on that, it is shown with a banner and the title, and with an hero image in the Bigpicture mode.

### Screenshots

![grafik](https://user-images.githubusercontent.com/68396806/232886003-deed839f-aa0d-4a20-af7b-f6d2b516787b.png)

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Ubuntu"]: Vanilla OS (based on Ubuntu)

### Additional information

Steam and hgl are installed via flatpak. I already created another issue for flatpak problems, these are solved now.

**Comment by Nocccer** [comment]
This depende if the game provides a icon. Mabye i miss something but if the game does not provide it we need to find another endpoint for it. Might be difficult

**Comment by Durus-R** [comment]
No, you are wrong, because I found it in the grid folder, while browsing the files.

**Comment by Nocccer** [comment]
> No, you are wrong, because I found it in the grid folder, while browsing the files.

I wrote the code. See above the code peace that sets the icon. There is no copy to grid.

**Comment by Nocccer** [comment]
I have the game so i will check later what happens there.

**Comment by Nocccer** [comment]
Mhh my logo looks good:
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/61798668/c2c7eb3f-ed10-4c84-85e1-dcfcac50df09)

I download the game from epic. Did you downloaded it from gog?

**Comment by Nocccer** [comment]
Icon provided by gamesdb is not better:
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/61798668/6f96d32b-6064-4b67-980c-7b4a0238d0f1)

**Comment by Nocccer** [comment]
Steamdb provides the icon but the image is behind a hash value which i don't know how to create.
For Bloons TD6: `https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/apps/960090/c5f0b67c8beeb6be99abe2ad8db2cada5d5ccff0.jpg`
